copy_input_files.sh

```diff
@@ -0,0 +1,53 @@
+set -e
+
+INPUT_BASE=/proj/ads/abstracts/config/
+TIMESTAMP=$(date +%Y-%m-%d_%H-%M-%S)
+OUTPUT_BASE=./logs/input/input.$TIMESTAMP/
+MIN_LINES=2400
```

Comment:
I imagine this was the minimum that would work for all the files, but it is quite far from some of the files. For instance:

```
sblanco@adsqb:~$ wc -l /proj/ads/abstracts/config/links/pub_pdf/all.links
1120305 /proj/ads/abstracts/config/links/pub_pdf/all.links
sblanco@adsqb:~$ wc -l /proj/ads/abstracts/config/links/eprint_pdf/all.links
2442485 /proj/ads/abstracts/config/links/eprint_pdf/all.links
```

I suggest we have a specific minimum per file so that we can catch more problems (e.g., now we would not catch if we lose half the lines in `pub_pdf`).